cg/meta/workflow/rnafusion.py

```diff
@@ -274,10 +283,19 @@ def run_analysis(
 
         else:
             LOG.info("Pipeline will be executed using tower")
-            parameters: List[str] = TowerAnalysisAPI.get_tower_launch_parameters(
-                tower_pipeline=self.tower_pipeline,
-                command_args=command_args,
-            )
+            if command_args.get("resume"):
```

Comment:
It would be great to make the commands_args into a pydantic model in the future to avoid knowing these keys.

Reply:
If it was a pydantic model you can write `command_args.resume` and the editor will know and supply the keys as the object is fully defined in the module, i.e. you do not have to know there is a "resume" key as the model will tell you there is 😄